utils/paginator.py

```diff
@@ -16,6 +17,14 @@
     from nextcord import Message
 
 
+def list_chunks(data: List[Any]) -> Generator[List[Any], Any, Any]:
```

Comment:
I'm curious as to why this was moved here. `player/__init__.py` is the only place this is used which is why I put it there, but now I'm thinking maybe we should move it to a new file in `utils` instead, since it is after all a plain util function. Would love to hear your thoughts 😃

Reply:
No, it looks good! Definitely belongs more in `utils` than in `player` 👍